(DOCSP-36170): Updated stem sentence guidance. (#166)
mongodb/docs-meta: source/style-guide/style/lists.txt

       period instead of a colon.

       .. note::

          Where possible, avoid using fragments to introduce lists. 

>           However, you may use stem sentences to avoid especially 

Reviewer: ```suggestion
          However, you may use stem sentences to avoid especially 
```
Would be nice to have an example of an appropriate stem sentence. Wasn't familiar with the term!

>           awkward or redundant constructions. Use your best discretion. 

Reviewer 2: Suggestion:

I think it would be helpful to continue to inform the reader why fragments should be avoided ("harder to understand than sentences"). I think the following phrasing might be more concise without losing any essential info:

"Avoid using fragments to introduce lists since they can be more difficult to understand than sentences. In cases in which sentences make the introduction awkward or redundant, you can use sentences stems."

>           Consider accessibility and screen readers.

Reviewer 2: Question:

I wasn't able to find any stand-out considerations in a quick search on screen readers. Could this sentence explain or link to some of the ones that should be kept in mind?

       period instead of a colon.

       .. note::

          Avoid using a fragment to introduce a list. Users might 
          find a fragment more difficult to understand than a full 
          sentence. If a full sentence makes the introduction awkward 

>           or redundant, you may use a stem sentence where the items in 
>           the list complete a full sentence. Consider accessibility and 

Reviewer 2: Suggestion:

I don't know if there's an official term, but I was only able to find references to the term "sentence stem", and not "stem sentence". I thought the direction of only using sentence stems if the list items already fit the sentence was the opposite of what I would try, which is to update the list items to complete the sentence. Here's how I would reword these lines:

```suggestion
          or redundant, use a sentence stem to introduce the list and
          make sure the list items complete the sentence. Consider accessibility and 
```
